src/components/case/CreateForm.vue

```diff
@@ -182,7 +210,9 @@ export default {
       category: "",
       isAnonymous: false,
       // mention: "",
-      tagadded: false,
+      tagAdded: false,
+      files: [],
+      proofs: [],
```

Comment:
This variable is not used anywhere

Reply:
Sir, it is used for selecting categories!